tests/jenkins/lib-testers/RunIntegTestScriptLibTest.groovy

```diff
@@ -0,0 +1,40 @@
+import static org.hamcrest.CoreMatchers.notNullValue
+import static org.hamcrest.MatcherAssert.assertThat
+
+
+class RunIntegTestScriptLibTester extends LibFunctionTester {
+
+    private String jobName
+    private String buildManifest
+    private String testManifest
+    private String buildId
+
+    public RunIntegTestScriptLibTester(jobName, buildManifest, testManifest, buildId){
+        this.jobName = jobName
+        this.buildManifest = buildManifest
+        this.testManifest = testManifest
+        this.buildId = buildId
+    }
+
+    void configure(helper, binding) {
+        // N/A
+    }
+
+    void parameterInvariantsAssertions(call) {
+        assertThat(call.args.jobName.first(), notNullValue())
```

Comment:
I think the job name can be null, since we substitute the default value for the jobname with `distribution-build-opensearch`